airflow/providers/amazon/aws/sensors/rds.py

```diff
@@ -61,7 +66,14 @@ def _check_item(self, item_type: str, item_name: str) -> bool:
         except ClientError:
             return False
         else:
-            return bool(items) and any(map(lambda s: items[0]['Status'].lower() == s, self.target_statuses))
+            status_field = self._check_status_field()
```

Comment:
2022/09/05 update (https://github.com/apache/airflow/pull/26003/commits/0d537b1bdf1fa3dbd8a94f1b635a5814e84e11e9)
Replaced instance variable with instance method to encapsulate if/else statement